src/primitives/transaction.h

```diff
@@ -452,14 +443,35 @@ class CTxCrosschainOut
 
     bool IsDust(const CFeeRate &minRelayTxFee) const
     {
-        return (nValue < GetDustThreshold(minRelayTxFee));
+        return (GetScValue() < GetDustThreshold(minRelayTxFee));
```

Comment:
For sure IsDust() is not to be used for mbtr, for the time being I will move IsDust() to fw/cr ccoutput base class, so that mbtr does not have it. Also, you are probably right and we do not need it at all for ccouts, but let us discuss it separatedly.

Reply:
After the discussion, we agreed to remove IsDust from ccoutput interface. The issue of having some minimum amount (expecially for sc creation) remains, but we will handle it differently.